src/jit/emit.h

```diff
@@ -1209,6 +1209,16 @@ class   emitter
         regMaskTP       idcGcrefRegs;              // ... gcref registers
         regMaskTP       idcByrefRegs;              // ... byref registers
         unsigned        idcArgCnt;                 // ... lots of args or (<0 ==> caller pops args)
+
+#ifdef FEATURE_MULTIREG_RET
+        GCtype          idSecondGCref() const { return (GCtype)_idcSecondRetRegGCType; }
+        void            idSecondGCref(GCtype gctype) { _idcSecondRetRegGCType = gctype; }
+#endif // FEATURE_MULTIREG_RET
+
+    private:
+#ifdef FEATURE_MULTIREG_RET
+        GCtype          _idcSecondRetRegGCType : 2;      // ... GC type for the second return register.  
```

Comment:
If it can't be put there, there should at least be comments (in both places) mentioning where the "other half" is.